projects/plugins/jetpack/_inc/lib/admin-pages/class-jetpack-redux-state-helper.php

```diff
@@ -211,44 +211,46 @@ public static function get_initial_state() {
 				'activationNoticeDismiss' => Licensing::instance()->get_license_activation_notice_dismiss(),
 			),
-			'jetpackManage'                        => array(
+			'jetpackManage'               => array(
 				'isEnabled'       => Jetpack_Manage::could_use_jp_manage(),
 				'isAgencyAccount' => Jetpack_Manage::is_agency_account(),
 			),
-			'hasSeenWCConnectionModal'             => Jetpack_Options::get_option( 'has_seen_wc_connection_modal', false ),
-			'newRecommendations'                   => Jetpack_Recommendations::get_new_conditional_recommendations(),
+			'hasSeenWCConnectionModal'    => Jetpack_Options::get_option( 'has_seen_wc_connection_modal', false ),
+			'newRecommendations'          => Jetpack_Recommendations::get_new_conditional_recommendations(),
 			// Check if WooCommerce plugin is active (based on https://docs.woocommerce.com/document/create-a-plugin/).
-			'isWooCommerceActive'                  => in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', Jetpack::get_active_plugins() ), true ),
-			'useMyJetpackLicensingUI'              => My_Jetpack_Initializer::is_licensing_ui_enabled(),
-			'isOdysseyStatsEnabled'                => Stats_Options::get_option( 'enable_odyssey_stats' ),
-			'shouldInitializeBlaze'                => Blaze::should_initialize(),
-			'isBlazeDashboardEnabled'              => Blaze::is_dashboard_enabled(),
-			'isSubscriptionSiteEnabled'            => apply_filters( 'jetpack_subscription_site_enabled', false ),
-			'newsletterDateExample'                => gmdate( get_option( 'date_format' ), time() ),
-			'subscriptionSiteEditSupported'        => $current_theme->is_block_theme(),
-			/* This filter is already documented in jetpack/modules/subscriptions.php */
-			'isWpAdminSubscriberManagementEnabled' => apply_filters( 'jetpack_wp_admin_subscriber_management_enabled', false ),
+			'isWooCommerceActive'         => in_array( 'woocommerce/woocommerce.php', apply_filters( 'active_plugins', Jetpack::get_active_plugins() ), true ),
+			'useMyJetpackLicensingUI'     => My_Jetpack_Initializer::is_licensing_ui_enabled(),
+			'isOdysseyStatsEnabled'       => Stats_Options::get_option( 'enable_odyssey_stats' ),
+			'shouldInitializeBlaze'       => Blaze::should_initialize(),
+			'isBlazeDashboardEnabled'     => Blaze::is_dashboard_enabled(),
+			'subscriptions'               => array(
+				'isSubscriptionSiteEnabled'            => apply_filters( 'jetpack_subscription_site_enabled', false ),
+				'subscriptionSiteEditSupported'        => $current_theme->is_block_theme(),
+				'newsletterDateExample'                => gmdate( get_option( 'date_format' ), time() ),
+				'isWpAdminSubscriberManagementEnabled' => apply_filters( 'jetpack_wp_admin_subscriber_management_enabled', false ),
+				'isFeaturedImagesInEmailsAutoEnabled'  => Automattic\Jetpack\Modules\Subscriptions\Settings::should_auto_enable_featured_images_emails(),
```

Comment:
Lots of whitespace changes in here, but these are the only relevant changes:

1. Moved the Subscription-related things into a single array so they're easier to find.
2. Add a call to the new method.